kafka-admin/.openapi/kafka-admin-rest.yaml

```diff
@@ -1146,6 +1186,8 @@ components:
       - groupId
       - consumers
       type: object
+      allOf:
```

Comment:
This is the source of the problem. 
It doesn't seem to look correct to me:

allOff: is aggregation mechanism to merge multiple objects. 
When used it should aggregate all properties/objects. 
In current form it is only containing single object - meaning that it is redundant.

Additionally when using allOff we should have all properties defined inside it. 
Having properties outside allOff,oneOff etc. creates non deterministic schema 
that fails under each individual generator implementation.
What I see is that most of the generators give up in this case and simply does use HashMap.
However golang generator has bug for that use case and still expects proper object.